composer.json

```diff
@@ -1,24 +1,41 @@
 {
-	"name" : "chillu/fakedatabase",
-	"description" : "Simple 'database' storage adapter which persists into a JSON file",
-	"license" : "BSD-2-Clause",
-	"keywords" : ["json","testing","database"],
-	"authors" : [
+	"name": "chillu/fakedatabase",
+	"description": "Simple 'database' storage adapter which persists into a JSON file",
+	"license": "BSD-2-Clause",
+	"keywords": ["json","testing","database"],
+	"authors": [
 		{
 			"name" : "Ingo Schommer",
 			"email" : "me@chillu.com",
 			"role" : "Developer"
+		},
+		{
+			"name": "madmatt",
+			"email": "signups+github@madman.net.nz"
 		}
 	],
+	"require": {
```

Comment:
Wanna set a branch-alias: 2.x as well? So nobody pulls in those breaking changes automatically

Reply:
Done :)